orangecontrib/bio/genialis/genapi.py

```diff
@@ -0,0 +1,190 @@
+from genesis import Genesis
+from genesis import GenData
+import json
+from Orange.data import *
+from .. import dicty
+import io
+import gzip
+
```

Comment:
Here should be 2 empty lines. PEP8 will report all this warnings, so I will ignore the others from now on.